stable_baselines/gail/dataset/record_expert.py

```diff
@@ -38,7 +38,6 @@ def generate_expert_traj(model, save_path, env=None, n_timesteps=0,
     is_vec_env = False
     if isinstance(env, VecEnv):
         is_vec_env = True
-        assert env.num_envs == 1, 'You must use only one env to record expert data'
```

Comment:
I would add a warning to tell the use that only one env will be used to record data

Reply:
good idea, done